### PR TITLE
Fix: bug of display list submitted ontologies of account setting

### PR DESCRIPTION
**Pull request by SirineMhedhbi**
## Context :
see : https://github.com/ontoportal-lirmm/bioportal_web_ui/issues/270

## Changes : 
- Display list of ontologies by acronym ( Ontologies + views of ontologies) in the liste of Submitted ontologies

image

### CODE REVIEW

**Review by syphax-bouazzouni — CHANGES_REQUESTED** [review]
Write a better PR name

File: app/views/users/show.html.haml [hunk 1]
```diff
@@ -113,6 +113,10 @@
               - @admin_ontologies.each do |ont|
                 .account-page-submitted-ontology{data: {controller: 'tooltip'}, title: ont.name}
                   %a{href: "/ontologies/#{ont.acronym}"}= ont.acronym
+                - unless ont.views.nil? || ont.views.empty?
+                  - ont.views.each do |view|
```
Comment (syphax-bouazzouni) [inline app/views/users/show.html.haml:117]: This solution is not good.

The goal is not to show the views of the submitted ontologies.

But to show all the submitted ontologies and views of the users (currently only the ontologies are listed)

**Review by syphax-bouazzouni — CHANGES_REQUESTED** [review]
Does not resolve the issue.
See https://github.com/ontoportal-lirmm/bioportal_web_ui/issues/270#issuecomment-1621848656

**Review by syphax-bouazzouni — APPROVED** [review]

File: app/views/users/show.html.haml (hunk 1 above)
Comment (SirineMhedhbi) [reply app/views/users/show.html.haml:117]: @SirineMhedhbi  remove this code and open a new PR

**Pull request merged**